src/components/screens/EventRecords.js

```diff
@@ -375,6 +393,20 @@ export const EventRecords = ({ navigation, route }) => {
         }
         showBackToTop
       />
+      {eventListIntro?.buttonType == EVENT_SUGGESTION_BUTTON.BOTTOM_FLOATING && (
+        <EventSuggestionButton
+          buttonTitle={eventListIntro.buttonTitle}
+          onPress={() => {
+            if (eventListIntro.url) {
+              openLink(eventListIntro.url, openWebScreen);
+            } else {
+              navigation.navigate(ScreenName.EventSuggestion, {
+                formIntroText: eventListIntro.formIntroText
+              });
+            }
+          }}
```

Comment:
the `onPress` is duplicated here, right? please create a callback function to have it in one place and call it for top and bottom-floating